src/View/Cell.php

```diff
@@ -302,12 +357,12 @@ public function __toString()
     public function __debugInfo()
     {
         return [
-            'plugin' => $this->plugin,
             'action' => $this->action,
             'args' => $this->args,
-            'template' => $this->template,
+            'viewClass' => $this->viewClass,
```

Comment:
Do Cells have viewClass anymore? I removed it in #12187

Reply:
`Cell` uses `ViewVarsTrait` and there's `ViewVarsTrait::$viewVars` property.

Reply:
But `ViewVarsTrait::$viewClass` is deprecated, so I'll remove it. Must have snuck in when I merged `3.next` into this branch.